data/json/effects.json

```diff
@@ -1665,5 +1665,44 @@
   {
     "type": "effect_type",
     "id": "has_prospectus"
+  },
+  {
+    "type": "effect_type",
+    "id": "rooted",
+    "//": "What keeps rooted plants from moving around.",
+    "name": [ "Rooted" ],
+    "desc": [ "You've sunk your roots into the ground, and are unable to move." ]
+  },
+  {
+    "type": "effect_type",
+    "id": "something",
```

Comment:
Doesn't look like you use these for anything.  Maybe make `"deficiency"` an optional field for vitamins?

Reply:
The game will complain if these are absent.

Reply:
Yes, and you could modify how vitamins are defined to make it not complain and remove this workaround.